.snap_goreleaser.yml

```diff
@@ -0,0 +1,11 @@
+project_name: dunner
+
+snapcraft:
+  name_template: '{{ .ProjectName }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+  summary: A Docker based task runner tool
+  description: |
+    Dunner is a task runner tool like Grunt but used Docker images like CircleCI do. | 
+    You can define tasks and steps of the tasks in your `.dunner.yaml` file and then run these steps with `Dunner do taskname`
+  grade: stable
+  confinement: classic
```

Comment:
Shall we try again with `strict` confinement?

Reply:
I updated with snap changes for strict. Please review now.